README.md

```diff
@@ -69,7 +69,12 @@ Apache Polaris is built using Gradle with Java 21+ and Docker 27+.
 - `./gradlew assemble` - To skip tests.
 - `./gradlew test` - To run unit tests and integration tests.
 - `./gradlew run` - To run the Polaris server locally; the server is reachable at 
-  localhost:8181. This is also suitable for running regression tests, or for connecting with Spark. 
+    localhost:8181. This is also suitable for running regression tests, or for connecting with Spark. 
+  - If you start Polaris as a direct java process via `./gradlew run`, you can set your own credentials using the environment variable `export POLARIS_BOOTSTRAP_CREDENTIALS=POLARIS,root,secret` where:
+    - `POLARIS` is the realm
+    - `root` is the CLIENT_ID
+    - `secret` is the CLIENT_SECRET
+  - If credentials are not set, it will use preset credentials `POLARIS,root,secret`
```

Comment:
The `POLARIS,root,secret` default applies only to `./gradlew run`

Reply:
This is fine, this part is only for `./gradlew run`